m4_sessions/1708-inning/group_one.md

```diff
@@ -1,9 +1,32 @@
 # Group One
 
 ### Session Title:
+* Transhumanism - 
 
 ### Session Outcomes: 
+* Create an understanding of what transhumanism is.
+* How does this relate to us as developers of software?
+* History, now, and future of transhumanism
+* Discussion humanism and see how people's feelings and thoughts change after the session.
 
 ### Session Outline:
+* Whiteboard spectrum of how people feel about this.
```

Comment:
Highly recommend you do a slack channel and send out a few resources ahead of time to give people a bit more time to determine how they feel.  Or you could have people go to corners like:  I understand fully and agree with it.  I don't know enough about it.  I think it sounds scary and I don't want to be a cyborg.